di/admin_units.py

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Thu Oct 22 17:53:54 2020
+
+@author: giuseppeperonato
+"""
+
+import geopandas as gpd
+import pandas as pd
+import sqlalchemy as sqla
+import utilities
+
+# GISCO datasets GEOJSON EPSG:4326 1:1milion
+datasets = {
+    "countries": "https://gisco-services.ec.europa.eu/distribution/v2/countries/geojson/CNTR_RG_01M_2020_{}.geojson",
+    "nuts": "https://gisco-services.ec.europa.eu/distribution/v2/nuts/geojson/NUTS_RG_01M_2021_{}.geojson",
+    "lau": "https://gisco-services.ec.europa.eu/distribution/v2/lau/geojson/LAU_RG_01M_2019_{}.geojson",
+}
+
+
+def get(datasets=datasets, crs="EPSG:4326"):
+    """
+    Retrieve NUTS, LAU and countries from GISCO API and make a single, consistent GDF.
+
+    Parameters
+    ----------
+    datasets : dict
+        dict with API URLs.
+    crs : TYPE, optional
+        Spatial Reference System. The default is "EPSG:4326".
+
+    Returns
+    -------
+    admin_units : GeoDataFrame
+        Table with all administrative units.
+
+    """
+    print("Downloading countries...")
+    countries = gpd.read_file(datasets["countries"].format(crs.split(":")[-1]), crs=crs)
```

Comment:
I feel like the crs parsing could be done by a lib, or at least the split could be separated high up in the function.